waspc/src/Wasp/Generator/WebAppGenerator/AuthG.hs

```diff
@@ -57,3 +57,8 @@ genUseAuth auth = return $ C.mkTmplFdWithData [relfile|src/auth/useAuth.ts|] tmp
   where
     tmplData = object ["entitiesGetMeDependsOn" .= makeJsArrayFromHaskellList [userEntityName]]
     userEntityName = AS.refName $ AS.Auth.userEntity auth
+
+genLogoutAction :: Generator FileDraft
+genLogoutAction = return $ C.mkTmplFdWithData (C.asTmplFile [relfile|src/auth/logout.ts|]) tmplData
+  where
+    tmplData = object ["logoutPath" .= ("/auth/logout" :: String)]
```

Comment:
How do you know this path? It is hardcoded only here. So you are basing it on the knowledge that is embedded in the rest of the Genreator/templates but is not hardcoded explicitly like this. What I guess I am trying to say is that I am not sure if you did much with this move here, it seems more reasonable to me to embed this directly into template instead of passing it via Haskell. But maybe I am missing something?

Reply:
Makes sense, if I connected the backend and frontend routes via Haskell, it would have made more sense. But I'll simplify 👍